pymc3/tests/test_distributions_random.py

```diff
@@ -1675,3 +1675,21 @@ def test_issue_3706(self):
             prior_pred = pm.sample_prior_predictive(1)
 
         assert prior_pred["X"].shape == (1, N, 2)
+
+
+def test_issue_3585():
```

Comment:
I would give this a more descriptive name and add a link in the doc-string/comments..